### PR TITLE
Add HCA publication email template in the SOPs

### PR DESCRIPTION
**Pull request by arschat**
Added also a no publication template by @Wkt8

Duplicate PR after accidental merge of ebi-ait/hca-ebi-wrangler-central#1060 before @ESapenaVentura review

### CODE REVIEW

File: docs/SOPs/contributor_communication_SOP.md [hunk 1]
```diff
@@ -146,6 +151,34 @@ These templates are considered a guide to provide a skeleton with the required i
...
+> Congratulations on your HCA manuscript! We are looking forward to hearing more about your project, and will add your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication once a preprint or publication is available. 
+> 
+> Do you have any time-sensitive requirements on archiving, or have you already archived your data in a specific database?
+> 
+> Warm regards,
+> [wrangler name] 
+> 
+> On Behalf of the HCA DCP Ingestion Service
+
+#### HCA branded publication - first contact
```
Comment (ESapenaVentura) [inline docs/SOPs/contributor_communication_SOP.md:167]:
```suggestion
#### HCA branded publication - with preprint or publication
```

File: docs/SOPs/contributor_communication_SOP.md [hunk 2]
```diff
@@ -43,6 +43,11 @@ The first contact email should contain:
 
 [Template email](#first-contact---data-questionnaire){: .btn .btn-blue }
 
+### Contact after an HCA publication email
+Wrangler team might be CC'ed in an email informing the authors that a publication has been accepted as an HCA publication (subject __HCA Branded Publication__). In this case, we can contact them and do not wait for them to reach. 
+
+[Template email](#hca-branded-publication---first-contact){: .btn .btn-blue }
```
Comment (ESapenaVentura) [inline docs/SOPs/contributor_communication_SOP.md:49]:
```suggestion
[Template email- no preprint or publication available](#hca-branded-publication---no-preprint-or-publication){: .btn .btn-blue }
[Template email- preprint or publication available](#hca-branded-publication---with-preprint-or-publication){: .btn .btn-blue }
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
Mostly LGTM, a comment on a specific part of the email (about archiving) and on naming of the email templates

**Review by idazucchi — APPROVED** [review]

File: docs/SOPs/contributor_communication_SOP.md [hunk 3]
```diff
@@ -146,6 +152,34 @@ These templates are considered a guide to provide a skeleton with the required i
...
+
+#### HCA branded publication - with preprint or publication
+
+> Dear [name/s of contributor/s],
+> 
+> Congratulations on your HCA manuscript!
+> We have added the information about your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication.
+> Due to internal processes it will take a day to display the publication information, in the meantime any feedback about the HCA project catalogue is greatly appreciated!
+> 
+> If you have already archived your data in any database, could you provide us the accession number? If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also, please inform us for any time-sensitive requirements on archiving.
```
Comment (ESapenaVentura) [inline docs/SOPs/contributor_communication_SOP.md:176]: I'd separate the 2 ideas here in 2 different paragraphs:
- First one, about asking for a pointer to the data
- Second one, offering to wrangle their data to the HCA data portal

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]
mostly LGTM, added a small comment on dividing a paragraph

File: docs/SOPs/contributor_communication_SOP.md [hunk 4]
```diff
@@ -146,6 +152,36 @@ These templates are considered a guide to provide a skeleton with the required i
...
+
+> Dear [name/s of contributor/s],
+> 
+> Congratulations on your HCA manuscript!
+> We have added the information about your publication to our project catalogue (https://www.ebi.ac.uk/humancellatlas/project-catalogue/) as a HCA publication.
+> Due to internal processes it will take a day to display the publication information, in the meantime any feedback about the HCA project catalogue is greatly appreciated!
+> 
+> If you have already archived your data in any database, could you provide us with the accession number?
+> 
+> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements on archiving.
```
Comment (idazucchi) [inline docs/SOPs/contributor_communication_SOP.md:178]:
```suggestion
> If you would like your data to be added to the HCA DCP Data Portal, please contact us and we can start the submission process. Also in that case, please inform us of any time-sensitive requirements.
```

Comment (idazucchi) [reply docs/SOPs/contributor_communication_SOP.md:178]: If they already deposited their data in an archive there is no need for us to offer archiving. 
We are unable to archive right now anyway so there is no point in mentioning it

**Pull request merged**